portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java

```diff
@@ -140,13 +140,17 @@ public void poll() {
     worker.doWork();
   }
 
-  private static List<TransferExtension> getTransferExtensions() {
+  private static List<TransferExtension> getTransferExtensions(Monitor monitor) {
```

Comment:
Merge error?

Reply:
No this is intentional, I had a bug where the plugin wasn't loading so I added this logging to help.